libs/prelude/Language/Reflection/Elab.idr

```diff
@@ -510,6 +540,17 @@ namespace Tactics
   defineFunction : FunDefn Raw -> Elab ()
   defineFunction defun = Prim__DefineFunction defun
 
+  ||| Declare a datatype in the global context. This step only
```

Comment:
Outside of elab is it possible to define a datatype e.g. in a where clause? If so, any reason this can't be scope-limited too?